connector/connect/src/main/scala/org/apache/spark/sql/connect/command/SparkConnectCommandPlanner.scala

```diff
@@ -79,6 +85,32 @@ class SparkConnectCommandPlanner(session: SparkSession, command: proto.Command)
     session.udf.registerPython(cf.getPartsList.asScala.head, udf)
   }
 
+  def handleCreateViewCommand(createView: proto.CreateDataFrameViewCommand): Unit = {
+    val viewType = if (createView.getIsGlobal) GlobalTempView else LocalTempView
+
+    val tableIdentifier =
+      try {
+        session.sessionState.sqlParser.parseTableIdentifier(createView.getName)
```

Comment:
do we need to parse it? It should be the plain view name.

Reply:
This is from Dataset API implementation and I guess it was to pre-validate if the view name is in a legal format.

If later there is a place to validate it then we can skip this.

Reply:
```
IDENTIFIER
    : (LETTER | DIGIT | '_')+
    ;
```

The passing in view name might contains more than the defined grammar above

Reply:
I see